createrepo_c.spec

```diff
@@ -89,7 +89,7 @@ These development files are for easy manipulation with a repodata.
 Summary:        Python 3 bindings for the createrepo_c library
 %{?python_provide:%python_provide python3-%{name}}
 BuildRequires:  python3-devel
-BuildRequires:  python3-nose
+BuildRequires:  python3-unittest2
```

Comment:
Hum. This package is only in F32. The package description says:

> unittest2 is a backport of the new features added to the unittest testing
> framework in Python 2.7 and onwards. It is tested to run on Python 2.6, 2.7,
> 3.2, 3.3, 3.4 and pypy.

Do we actually need this? If yes, it would appear the require needs to be conditional.

Reply:
Ok, I think this can be removed. Thanks.